osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -505,13 +505,15 @@ private Vector2 computePositionInStack(Vector2 position, float displayRadius)
             return position;
         }
 
-        private void addLighting(CatchHitObject hitObject, float x, Color4 colour)
+        private void addLighting(PalpableCatchHitObject hitObject, float catchPosition, Color4 colour, JudgementResult result)
         {
-            HitExplosion hitExplosion = hitExplosionPool.Get();
+            PoolableHitExplosion hitExplosion = hitExplosionPool.Get();
             hitExplosion.HitObject = hitObject;
-            hitExplosion.X = x;
-            hitExplosion.Scale = new Vector2(hitObject.Scale);
+            hitExplosion.CatchPosition = catchPosition;
             hitExplosion.ObjectColour = colour;
+            hitExplosion.JudgementResult = result;
+            hitExplosion.CatcherWidth = catchWidth;
+            hitExplosion.CatcherMargin = (1 - ALLOWED_CATCH_RANGE) / 2;
```

Comment:
sounds like it should just be a constant in `Catcher` or `LegacyCatchHitExplosion`?